packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -374,6 +374,9 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
 
     await this.build(options)
 
+    // Copy static assets after build completes
+    await this.copyStaticAssets()
+
```

Comment:
This is very specific to UI for now, and if we ever have a generalized solution it won't be using this method.   
  
Let's move this call to be executed after line 355, in the `build`​ function.